src/app/extensions/quoting/pages/quote/quote-page.component.html

```diff
@@ -14,15 +14,20 @@ <h1 [ngSwitch]="true">
     </h1>
 
     <ish-error-message [error]="error$ | async" />
+    <ng-container *ngIf="quote.items?.length; else emptyQuote">
```

Comment:
this way you don't catch all occurences:
<img width="1195" height="664" alt="grafik" src="https://github.com/user-attachments/assets/31483f64-00ff-420a-a26e-ed12b5de3058" />

remove the condition <ish-quote-line-item-list *ngIf="quote.items" /> everywhere and let the quote-items-list handle the display of the empty message